src/prometheus_alert_rules/hydra_slow_request_handling.rule

```diff
@@ -0,0 +1,8 @@
+alert: HydraSlowRequestHandling
+expr: histogram_quantile(0.9, sum(rate(http_requests_duration_seconds_bucket[5m])) by (le)) > 0.4
```

Comment:
as much as this is a good metric to chuck in here, we have no clue about response profiling so not sure how valuable an alert would be when the p90 is set to a random number